### DISCUSSION TITLE
Hide cannon entity hider

### DISCUSSION THREAD

**Discussion by Viking-Maori** [runelite/runelite#16717] [Ideas]
Hide cannon entity hider option would be great for new release events, for example trying out the new forestry event people setting up cannons obviously to mess with others, in general I think it would be ideal for boss events like Corp and accidentally click on other people's cannons.

**Comment by Nightfirecat** [comment]
We'd be fine with potentially expanding the entity hider plugin to allow customizable hides (similar to menu entry swapper's customizable left- and shift-click options), but need some way to be able to un-hide something after the fact.

**Reply by CreativeTechGuy** [reply]
A few ideas:

* Easier: A panel in the settings pane which lists the entities that you've hidden and you can remove them from the list.
* Harder but more Runelite-y: Hold alt and any hidden entities on screen will appear enabling you to click on them to unhide. (Similar to hidden ground items)
  * This should probably include a setting for clearing all hidden entities in case of situations where above doesn't work: accessibility and entities which are too small/fast to reliably click on.

**Comment by Nightfirecat** [comment]
For now, you can use the [Cannon Be Gone](https://runelite.net/plugin-hub/show/cannon-be-gone) hub plugin
